Fix conv2d bias INT48 type hint removed by fusing placeholder nods

3l1 opened the pull request:
Summary: Fix conv2d bias INT48 type hint removed by fusing placeholder nods

Reviewed By: Ninja91

Differential Revision: D84948170

3l1 commented:
@pytorchbot label "release notes: none"

digantdesai reviewed (approved):
Good debugging @3l1.

CC: @per as FYI

3l1 committed:
Fix conv2d bias INT48 type hint removed by fusing placeholder nods (#15257)

Summary:

Fix conv2d bias INT48 type hint removed by fusing placeholder nods

bypass-github-export-checks
bypass-github-pytorch-ci-checks
bypass-github-executorch-ci-checks

Reviewed By: Ninja91, digantdesai

Differential Revision: D84948170
